e2e/test/MultiplexingOverAmqp.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Azure.Devices.Client;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public static class MultiplexingOverAmqp
+    {
+        public const int MuxWithoutPoolingDevicesCount = 2;
+        public const int MuxWithoutPoolingPoolSize = 1; // For enabling multiplexing without pooling, the pool size needs to be set to 1
+        public const int MuxWithPoolingDevicesCount = 4;
```

Comment:
Maybe add a comment that these numbers will not be honored (and track with an Issue#)?